README.md

```diff
@@ -22,19 +22,19 @@ _Some similar features might have been implemented/merged_
 - Yarn/npm in favour of bower
   - Load only moment.js without languages (Languages are included in translations)
   - unneeded components removed (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
-- RBush v2 - performance boost in last versions (Positions labels and clients on the map)
+- RBush v2 - performance boost in last versions (positions, labels and clients on the map)
 - Ruby dependency removed
 - FixedCenter is required
-- Sass-lint, scss and variables rewritten for easy customization/adjustments
+- Sass-lint, scss and variables rewritten for easy customizations/adjustments
```

Comment:
Making it plural is ok, but it was singular on purpose.
German example: `für einfache Anpassung` vs `für einfache Anpassungen`
Both work, but I like the singular much better. It sounds cleaner. The plural is already given in `adjustments` making it clear there are multiple things one can do, so I don't think we need it there.